src/Libraries/CoreNodes/Types.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace DSCore
 {
+    [SupressImportIntoVM]
     public class Types
```

Comment:
DSCore.Types in UnitsUI.cs file will be inaccessible due to "Types" protection level if change attribute to internal.
@sharadkjaiswal